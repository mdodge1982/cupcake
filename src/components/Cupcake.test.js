import React from 'react';
import renderer from 'react-test-renderer';
import Cupcake from './Cupcake';

const cake = {
	frosted:'',
	yPos:134,
	xPos:345,
	width:42,
	height:56,
	wait:0
};

test('Initial render', () => {
	const component = renderer.create(
		<Cupcake {...cake} />
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});

test('Render with wait (after frosting hit)', () => {
	cake.wait = 18;
	const component = renderer.create(
		<Cupcake {...cake} />
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
