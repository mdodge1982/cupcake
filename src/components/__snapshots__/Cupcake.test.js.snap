// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initial render 1`] = `
<li
  className="Cupcake "
  style={
    Object {
      "bottom": "134px",
      "height": "56px",
      "left": "345px",
      "width": "42px",
    }
  }
>
  
</li>
`;

exports[`Render with wait (after frosting hit) 1`] = `
<li
  className="Cupcake "
  style={
    Object {
      "bottom": "134px",
      "height": "56px",
      "left": "345px",
      "width": "42px",
    }
  }
>
  <span
    className="burst"
  />
</li>
`;
